sdk/python/jobs/pipelines/1e_pipeline_with_registered_components/pipeline_with_registered_component_in_registry.ipynb

```diff
@@ -0,0 +1,248 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "b02e37bd",
+   "metadata": {},
+   "source": [
+    "# 1. Connect to Azure Machine Learning Workspace"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ae915f2e",
+   "metadata": {},
+   "source": [
+    "## 1.1 Import the required libraries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6bebb0a7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azure.identity import DefaultAzureCredential, InteractiveBrowserCredential\n",
+    "\n",
+    "from azure.ai.ml import MLClient\n",
+    "from azure.ai.ml.dsl import pipeline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "54eaf508",
+   "metadata": {},
+   "source": [
+    "## 1.2 Configure credential"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "463f5692",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "try:\n",
+    "    credential = DefaultAzureCredential()\n",
+    "    # Check if given credential can get token successfully.\n",
+    "    credential.get_token(\"https://management.azure.com/.default\")\n",
+    "except Exception as ex:\n",
+    "    # Fall back to InteractiveBrowserCredential in case DefaultAzureCredential not work\n",
+    "    credential = InteractiveBrowserCredential()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fa8c5101",
+   "metadata": {},
+   "source": [
+    "## 1.3 Get a handle to the workspace and the registry"
```

Comment:
>Get a handle to the workspace and the registry" [](http://example.com/codeflow?start=12&length=47)

Add detail to request user to provide sub id, rg, ws

Reply:
Adding support to extract it from config file, which will be used in workflow,adding description related to config file

Reply:
Added,